CHANGELOG.md

```diff
@@ -78,6 +78,7 @@
 - Add highlight when newly scanned bp passports are added
 - Index `patientUuid` in tables that are joined in `OverdueAppointment` database view
 - Migrate `RegistrationPinScreen` to use view effects
+- Fix broken back button in `EditPatientScreen`
```

Comment:
Ideally this should be in the fixes section, since it's in demo and we are hotfixing it. But I am merging this in for now.

Reply:
umm but this is not out there to the users yet so them seeing this as a fix would not make sense. @msasikanth

Reply:
We caught this bug internally in a pre-production environment which is why I thought we can have this in the internal section

Reply:
Yeah, that's fair. So far, if it goes into demo and we have to fix an bug fix, we place it in `Fixes` section of the changelog.